docs/integrations/webhooks/create-a-webhook.mdx

```diff
@@ -0,0 +1,361 @@
+3. To deploy your local server to the internet, run the following command in your terminal:
+
+`ngrok http http://localhost:3000`
+
+4. In your terminal, ngrok should display a **Forwarding** URL that looks something like `https://fb3f-2603-1700-2458-3410-a006-dc38-e9d6-1275.ngrok-free.app/`. Copy this URL.
+
+## Create an endpoint in the Clerk Dashboard
+
+To create a webhook endpoint, you must provide the **Endpoint URL** and then choose the events you want to listen to. For this guide, you will listen to the `user.updated` event.
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=webhooks).
+2. In the navigation sidenav, select **Webhooks**.
+3. Select the **Add Endpoint** button.
+4. In the **Endpoint URL** field, paste the ngrok URL you copied earlier followed by `/api/webhooks`. This is the endpoint that you will later create, that Svix will send the webhook payload to. The full URL should look something like `https://fb3f-2603-1700-2458-3410-a006-dc38-e9d6-1275.ngrok-free.app/api/webhooks`.
+5. In the **Message Filtering** section, select the `user.updated` event.
+6. Select the **Create** button.
+7. You will be redirected to your endpoint's settings page. Leave this page open.
+
+## Add your signing secret to your `.env` file
+
+To verify the webhook payload, you will need your endpoint's signing secret. However, you do not want to expose this secret in your codebase, so you will store it in a `.env` file.
+
+1. On the endpoint's settings page, copy the **Signing Secret**. It should be on the right hand side of the page with an eye icon next to it.
+2. In your project's root directory, you should have an `.env.local` file that includes your Clerk API keys. Here, assign your signing secret to `WEBHOOK_SECRET`. Your file should look something like this:
+
+<InjectKeys>
+
+```sh filename=".env.local"
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+CLERK_SECRET_KEY={{secret}}
+WEBHOOK_SECRET=whsec_123
+```
+
+</InjectKeys>
+
+## Install `svix`
+
+You will use [`svix`](https://www.npmjs.com/package/svix) to verify the webhook signature. Install it by running the following command in your terminal:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+  ```bash filename="terminal"
+  npm install svix
+  ```
+
+```bash filename="terminal"
+yarn add svix
+```
+
+```bash filename="terminal"
+pnpm add svix
+```
+
+</CodeBlockTabs>
+
+## Create a webhook handler
+
+Create a Route Handler that uses `svix` to verify the webhook signature and that receives the webhook's payload.
+
+For the sake of this guide, you will only log the payload to the console. In a real world application, you would use the payload to trigger some action. For example, you are listening for the `user.updated` event, so you could display a notification to all users in the application that a user was just updated.
```

Comment:
```suggestion
For the sake of this guide, you will only log the payload to the console. In a real world application, you would use the payload to trigger some action. For example, you are listening for the `user.updated` event, so you could perform a database `update` or `upsert` to update the user's details.
```

Not a required change, but probably reflective of the most common use case for user.updated.